### PR TITLE
add-epsilon-decreasing docs

### PR DESCRIPTION
**Pull request by imaimai1125**
Ref https://github.com/jubatus/jubatus_core/pull/381

### CODE REVIEW

File: source/en/api/api_bandit.rst [hunk 1]
```diff
@@ -55,6 +57,19 @@ We show each field below:
 
         * range of value: 0 <= ``seed`` <= :math:`2^{32} - 1`
 
+   epsilon_decreasing
+    :decreasing_rate:
+        Decreasing parameter of epsilon. The bigger this parameter becomes, the more slowly epsilon dcreases. (Float)
```
Comment (kmaehashi) [inline source/en/api/api_bandit.rst]: typo: dcreases

File: source/ja/api/api_bandit.rst [hunk 2]
```diff
@@ -53,7 +55,22 @@ JSON の各フィールドは以下のとおりである。
         (Integer)
 
         * 値域: 0 <= ``seed`` <= :math:`2^{32} - 1`
+  
+   epsilon_decreasing
+    :decreasing_rate:
+        epsilon_greedyで用いられているepsilonの値を試行回数が増えるごとに減らしていく、その割合。
```
Comment (TkrUdagawa) [inline source/ja/api/api_bandit.rst]: パラメータepsilon の話は直接epsilon decreasingと関わってきていないのでちょっと変な感じがしました。
「腕をランダムに選択する確率を減衰させる割合を指定する。」
ぐらいでどうでしょうか。

File: source/en/api/api_bandit.rst (hunk 1 above)
Comment (TkrUdagawa) [reply source/en/api/api_bandit.rst]: As commented in Japanese docs, I think `epsilon` is strange.
How about like below?
"Decreasing rate for the probability of selecting arms randomly. The bigger this parameter is, the more slowly the probability decreases."

File: source/en/api/api_bandit.rst [hunk 3]
```diff
@@ -55,6 +57,19 @@ We show each field below:
...
+
+        * Range: 0 < ``decreasing_rate`` < 1
+    
+    :seed(optional):
+        Specify random seed when you use ``random`` for unlearning strategy.
```
Comment (TkrUdagawa) [inline source/en/api/api_bandit.rst]: This parameter is not for unlearning.
The same mistakes are found in other methods.

File: source/ja/api/api_bandit.rst [hunk 4]
```diff
@@ -16,15 +16,17 @@ JSON の各フィールドは以下のとおりである。
...
+      設定値                     手法
+      ========================= ===================================
+      ``"epsilon_greedy"``      epsilon-greedy法を利用する。
+      ``"epsilon_decdeasing"``` Greedy Mix法 [#]_ を利用する 
+      ``"ucb1"``                UCB1法を利用する。
+      ``"softmax"``             softmax法を利用する。
+      ``"exp3"``                exp3法を利用する。
+      ``"ts"``                  Thompson samplingを利用する [#]_ 
+      ========================= ===================================
+   .. [#] `Greedy Mix <https://pdfs.semanticscholar.org/2c50/eccfb2878cc7e4286791f582f96c52a26da9.pdf>`_
```
Comment (TkrUdagawa) [inline source/ja/api/api_bandit.rst]: I think making algorithms page and linking there is better.

**Comment by TkrUdagawa** [comment]
Thanks

**Pull request merged**